<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Информация</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_title">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>113</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Title</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_author">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>113</width>
             <height>25</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>Author</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_year">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>113</width>
             <height>25</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>Year</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>30</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
           <property name="title">
            <string>Формат</string>
           </property>
           <widget class="QRadioButton" name="radioButton_isbook">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>112</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Книга</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_isjourn">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>112</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Журнал</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_show_prev">
          <property name="text">
           <string>&lt;&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_show_next">
          <property name="text">
           <string>&gt;&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Данные о книге</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_titl_in">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>113</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Title</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_author_in">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>113</width>
             <height>25</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>Author</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_year_in">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>113</width>
             <height>25</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>Year</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>30</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
           <property name="title">
            <string>Формат</string>
           </property>
           <widget class="QRadioButton" name="radioButton_book_in">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>112</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Книга</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_journ_in">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>112</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Журнал</string>
            </property>
           </widget>
          </widget>
          <widget class="QPushButton" name="pushButton_addBook">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>130</y>
             <width>89</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Режим</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
